gmail/src/services/odoo_auth.ts

```diff
@@ -130,3 +130,28 @@ export const resetAccessToken = () => {
     const userProperties = PropertiesService.getUserProperties();
     userProperties.deleteProperty("ODOO_ACCESS_TOKEN");
 };
+
+/**
+ * Ping the Odoo database to know if the "mail_plugin"
+ * is installed and if the URL is valid.
+ *
+ * Returns True is the URL is valid otherwise returns False.
```

Comment:
It's a little more than just the URL being valid/invalid, it's also about the server being accessible and the plugin being installed from what I understand.

Also, like in the outlook plugin, would it be possible to specify which route we are using for the check? The code makes it a but obscure.

Reply:
I'll rename it to `isOdooDatabaseReachable` so it match the outlook name